package.json

```diff
@@ -33,6 +33,13 @@
       "browser": true
     }
   },
+  "prettier": {
+    "bracketSpacing": false,
+    "printWidth": 80,
+    "proseWrap": false,
+    "semi": false,
+    "singleQuote": true
```

Comment:
These are the options that are used for Markdown and the code inside the fences. For more options check the full list: https://prettier.io/docs/en/options.html